src/components/MultiHopTrade/components/RateGasRow.tsx

```diff
@@ -75,22 +74,15 @@ export const RateGasRow: FC<RateGasRowProps> = memo(
     const [hasClickedRate, setHasClickedRate] = useState(false)
     const buyAsset = useAppSelector(state => selectAssetById(state, buyAssetId))
     const sellAsset = useAppSelector(state => selectAssetById(state, sellAssetId))
-    const foxBalanceCryptoPrecision = useAppSelector(state =>
-      selectVotingPower(state, { feeModel: 'SWAPPER' }),
-    )
 
     const feeMessage = useMemo(() => {
-      if (!affiliateBps || bnOrZero(affiliateBps).isZero())
-        return translate('trade.freeTrade', {
-          foxBalance: bnOrZero(foxBalanceCryptoPrecision).toFixed(0),
-        })
+      if (bn(affiliateBps).isZero()) return translate('trade.freeTrade')
```

Comment:
We are still showing this, which seems to imply that the trade if free because of FOX somehow.
We probably want to kill the "Learn more about FOX benefits" part.

<img width="1051" alt="Screenshot 2025-05-14 at 5 17 36 pm" src="https://github.com/user-attachments/assets/58ceaaf5-89b3-4949-b7fe-f1e311e38cc6" />

Reply:
Nice catch! Nuked in 05a7f33ad3 / fb18653754

<img width="1052" alt="image" src="https://github.com/user-attachments/assets/c8d62bd2-7b14-4c9a-ae61-9a446256334d" />